Frameworks/Foundation/NSHTTPURLResponse.mm

```diff
@@ -19,164 +19,116 @@
 #include "Foundation/NSHTTPURLResponse.h"
 #include "CoreFoundation/CFDictionary.h"
 #include "NSURLResponseInternal.h"
+#include "StubReturn.h"
 
-Boolean CFHTTPHeaderEqual(const void* obj1, const void* obj2) {
+static Boolean CFHTTPHeaderEqual(const void* obj1, const void* obj2) {
     return [(id)obj1 caseInsensitiveCompare:(NSString*)obj2] == 0;
 }
 
-CFHashCode CFHTTPHeaderHash(const void* obj1) {
+static CFHashCode CFHTTPHeaderHash(const void* obj1) {
     return [[(id)obj1 lowercaseString] hash];
 }
 
 @implementation NSHTTPURLResponse {
```

Comment:
#Pending

---
In reply to: [69179209](https://github.com/Microsoft/WinObjC/pull/595#discussion_r69179209) [](ancestors = 69179209,69179019,69176268)